packit_service/service/app.py

```diff
@@ -48,17 +48,19 @@ def get_flask_application():
     app = Flask(__name__)
     app.register_blueprint(blueprint)
     app.register_blueprint(builds_blueprint)
-    s = ServiceConfig.get_service_config()
+    service_config = ServiceConfig.get_service_config()
     # https://flask.palletsprojects.com/en/1.1.x/config/#SERVER_NAME
     # also needs to contain port if it's not 443
-    app.config["SERVER_NAME"] = s.server_name
+    app.config["SERVER_NAME"] = service_config.server_name
     app.config["PREFERRED_URL_SCHEME"] = "https"
     if getenv("DEPLOYMENT") in ("dev", "stg"):
         app.config["DEBUG"] = True
+    app.config["DASHBOARD"] = service_config.dashboard_url
```

Comment:
nit: I'd name it DASHBOARD_URL :)

Reply:
thx for noticing, I can remove it, it is loaded in `service.urls` as global constant :D this one is no longer needed